test/TimeTracker.Library.Test/Services/TimeEntryServiceTest.cs

```diff
@@ -3,30 +3,31 @@
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.EntityFrameworkCore;
-using TimeTracker.Library.Services;
 using TimeTracker.Data;
 using TimeTracker.Data.Models;
+using TimeTracker.Library.Services;
 using Xunit;
 
 namespace TimeTracker.Library.Test.Services
 {
     public class TimeEntryServiceTest : IClassFixture<InMemoryDatabaseWithProjectsAndUsers>
     {
         private readonly Guid userId = Guid.NewGuid();
-        
+
         private readonly TimeTrackerDbContext database;
         private readonly TimeEntryService timeEntryService;
 
-        public TimeEntryServiceTest(InMemoryDatabaseWithProjectsAndUsers inMemoryDatabase)
+        public TimeEntryServiceTest()
         {
+            var inMemoryDatabase = new InMemoryDatabaseWithProjectsAndUsers();
```

Comment:
I didn't do that in another test class, but just thought about this:
the InMemory... class needs to be disposed after each usage. (for releasing resources efficiently)

What to do:
make this test-class implement IDisposable (also, remove the IClassFixture) and implement the Dispose method to call something like this:
```C#
if (inMemoryDatabase != null) {
    inMemoryDatabase.Dispose();
}
```